kataloger/tigersys/sykmelding.bru

```diff
@@ -22,10 +22,10 @@ vars:pre-request {
 
 script:pre-request {
   const http = require('axios');
-  const responseSystembruker = await http.post('https://hag-lps-api-client.ekstern.dev.nav.no/systembruker', 'orgnr=311910663', {
-    headers: { 'Content-Type': 'application/x-www-form-urlencoded' }
-  });
+  let response = await http.post(
```

Comment:
Hva er årsaken til bytte til let? Er det ikke bedre å bruke imutable?

Reply:
Åja så det er kopiert fra en annen bru fil 😄. Skjønner
Jeg tenker at det er bedre å bruke const hvis man ikke trenger at det skal være mutable.
Hva tenker du @Jesper-Hustad ?

Reply:
Høres bra ut for meg - jeg bytter til const overalt 👍